website/requirements.txt

```diff
@@ -39,6 +39,7 @@ pytz==2024.1
 PyYAML==6.0.1
 requests==2.31.0
 rsa==4.9
+setuptools
```

Comment:
this does not have a version number, intentional? @OchiengPaul442

Reply:
If no version number is specified, the latest stable version of the package will be used. This ensures that the most up-to-date, stable version of the package is installed.